src/main/java/com/impactupgrade/integration/hubspot/ContactsClient.java

```diff
@@ -1,30 +1,40 @@
 package com.impactupgrade.integration.hubspot;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import com.impactupgrade.integration.hubspot.builder.ContactBuilder;
 import com.impactupgrade.integration.hubspot.exception.DuplicateContactException;
 import com.impactupgrade.integration.hubspot.exception.HubSpotException;
 import com.impactupgrade.integration.hubspot.model.Contact;
+import com.impactupgrade.integration.hubspot.model.ContactRequest;
 import com.impactupgrade.integration.hubspot.model.internal.ErrorResponse;
 
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class ContactsClient extends AbstractClient {
 
+  private final static Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+
   private final WebTarget contactsTarget = target.path("contacts/v1");
 
   public ContactsClient(String apiKey) {
     super(apiKey);
   }
 
   public Contact getByEmail(String email) {
-    return contactsTarget
+    Contact c = contactsTarget
         .path("contact/email/" + email + "/profile")
         .queryParam("hapikey", apiKey)
         .request(MediaType.APPLICATION_JSON)
         .get(Contact.class);
+    LOGGER.log(Level.INFO, c.toString());
```

Comment:
Unfortunately need to remove this, and carefully isolate that sort of logging to callers of this method outside this project. Otherwise, logging PII within a public, open source project (GDPR risk for users)